### PR TITLE
fix translate

### PR DESCRIPTION
**Pull request by chjxz**
图像增强 -> 图像增广
样式转移 -> 风格迁移

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-814/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-814/

File: chapter_computer-vision/index.md [hunk 1]
```diff
@@ -7,8 +7,8 @@
 有鉴于此，本章将重点介绍计算机视觉领域，并探讨最近在学术界和行业中具有影响力的方法和应用。 
 
 在 :numref:`chap_cnn` 和 :numref:`chap_modern_cnn` 中，我们研究了计算机视觉中常用的各种卷积神经网络，并将它们应用到简单的图像分类任务中。
-本章开头，我们将介绍两种可以改进模型泛化的方法，即 *图像增强* 和 *微调*，并将它们应用于图像分类。
-由于深度神经网络可以有效地表示多个层次的图像，因此这种分层表示已成功用于各种计算机视觉任务，例如 *对象检测*、*图像语义分割* 和 *样式转移*。
+本章开头，我们将介绍两种可以改进模型泛化的方法，即 *图像增广* 和 *微调*，并将它们应用于图像分类。
+由于深度神经网络可以有效地表示多个层次的图像，因此这种分层表示已成功用于各种计算机视觉任务，例如 *对象检测*、*图像语义分割* 和 *风格迁移*。
```
[the remark this answers is not in the document]
Comment (chjxz) [reply chapter_computer-vision/index.md]: I changed it

**Comment by chjxz** [comment]
It is my pleasure can join in this open source project

**Comment by mli** [comment]
Job d2l-zh/PR-814/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-814/

**Comment by mli** [comment]
Job d2l-zh/PR-814/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-814/

File: chapter_computer-vision/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (goldmermaid) [reply chapter_computer-vision/index.md]: Fabulous!

**Review by goldmermaid — APPROVED** [review]

**Pull request merged**